implementations/contracts/ERC725XCore.sol

```diff
@@ -49,6 +49,29 @@ abstract contract ERC725XCore is OwnableUnset, ERC165, IERC725X {
         return _execute(operationType, to, value, data);
     }
 
+    /**
+     * @inheritdoc IERC725X
+     */
+    function execute(
+        uint256[] memory operationsType,
+        address[] memory to,
+        uint256[] memory values,
+        bytes[] memory data
+    ) public payable virtual onlyOwner returns (bytes[] memory result) {
+        if (
+            operationsType.length != to.length ||
+            (to.length != values.length || values.length != data.length)
+        ) revert ERC725X_ExecuteParametersLengthMismatch();
+
+        result = new bytes[](operationsType.length);
+        for (uint256 i = 0; i < operationsType.length; i++) {
```

Comment:
`operationsType.length` may be cheaper to store in a memory variable? Wdyt? @CJ42 @b00ste

Reply:
*specially since we are also using this value on line 66 and 62

Reply:
Actually, it was more expensive by `1 gas unit`.
Without caching: 122176
With caching: 122177